### ISSUE TITLE
ANDROID_HOME

### ISSUE THREAD

**Issue by ebendutoit** [apstndb/zetasql-format-server#5]
Hi there. 

Building the latest version of this repo that uses the June 2020 zetasql fails with:

```
ERROR: Analysis of target '@com_google_zetasql//zetasql/public:sql_formatter' failed; build aborted: Either the path attribute of android_sdk_repository or the ANDROID_HOME environment variable must be set.
```

Have you already resolved this?

**Comment by apstndb** [comment]
It seems because `Dockerfile` use latest version of bazel and the recent version of bazel requires `ANDROID_HOME`.

https://github.com/google/zetasql#how-to-build
>ZetaSQL uses bazel for building and dependency resolution. After installing bazel (we maintain support for 1.0, but other versions may work), simply run:

ZetaSQL seems to support Bazel 1.0.x so I have fixed to `bazel-1.0.1` and it seems resolved.

**Comment by ebendutoit** [comment]
Wonderful!
